src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -52,18 +54,22 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
         	
         DmlWrapper dmlWrapper = new DmlWrapper();        
         list<Contact> listConAddrSet = new list<Contact>(); 
-        list<Contact> listConCreateAddr = new list<Contact>();
+        Set<Contact> setConCreateAddr = new Set<Contact>();
```

Comment:
@davidhabib per our discussion, changed the variable to a set and converted it back before calling the method.